libbpfgo/selftest/perfbuffers/run.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+GREEN='    \033[0;32m'
+RED='    \033[0;31m'
+NC='\033[0m' 
+
+VERSION_LIMIT=5.8
+CURRENT_VERSION=$(uname -r | cut -d '.' -f1-2)
+
+# Check major version
+MAJOR_VERSION_LIMIT=$(echo $VERSION_LIMIT | cut -d '.' -f1)
+MAJOR_CURRENT_VERSION=$(echo $CURRENT_VERSION | cut -d '.' -f1)
+if (( $(echo "$MAJOR_CURRENT_VERSION < $MAJOR_VERSION_LIMIT") |bc -l)); then
+    echo -e "${RED}[*] OUTDATED MAJOR KERNEL VERSION${NC}"
+    exit 1
+fi
+
+# Check minor version
+MINOR_VERSION_LIMIT=$(echo $VERSION_LIMIT | cut -d '.' -f2)
+MINOR_CURRENT_VERSION=$(echo $CURRENT_VERSION | cut -d '.' -f2)
+if (( $(echo "$MINOR_CURRENT_VERSION < $MINOR_VERSION_LIMIT") |bc -l)); then
+    echo -e "${RED}[*] OUTDATED MINOR KERNEL VERSION${NC}"
+    exit 0
+else
+    echo -e "${GREEN}[*] SUFFICIENT KERNEL VERSION${NC}"
+fi
```

Comment:
```suggestion
# Check minor version, only if MAJOR_CURRENT_VERSION is same as MAJOR_VERSION_LIMIT
MINOR_VERSION_LIMIT=$(echo $VERSION_LIMIT | cut -d '.' -f2)
MINOR_CURRENT_VERSION=$(echo $CURRENT_VERSION | cut -d '.' -f2)
if (( $(echo "$MAJOR_CURRENT_VERSION == $MAJOR_VERSION_LIMIT") |bc -l)); then
    if (( $(echo "$MINOR_CURRENT_VERSION < $MINOR_VERSION_LIMIT") |bc -l)); then
        echo -e "${RED}[*] OUTDATED MINOR KERNEL VERSION${NC}"
        exit 1
    else
        echo -e "${GREEN}[*] SUFFICIENT KERNEL VERSION${NC}"
    fi
fi
```

Reply:
@yanivagman @eyakubovich 

How does this look? We only need to check minor version if the major version is equal to the major version limit.

Reply:
Yes, I copied the whole file (as well as most of the files in the directory) as is from the ringbuffer test and didn't look too much into it. Since we're relying on `bc` anyway, you can just compare both major and minor in one shot:
```
if [ $(echo "$CURRENT_VERSION < $VERSION_LIMIT" | bc) -eq "1" ]; then
    echo -e "${RED}[*] OUTDATED KERNEL VERSION${NC}"
    exit 1
fi
```
Also perf buffer was added in 4.3 so I need to adjust the `VERSION_LIMIT` value.

Reply:
I'm not sure bc is good for version comparison:
```
❯ echo "1.30 < 1.3" | bc
0
```
I'd recommend `sort -v` which is version aware comparisons

Reply:
Got it, that works too, once you push that change i'll go ahead and approve!

Reply:
@itaysk you're right, of course, can't compare them as decimals. How do I do it with sort? All I can come up with is:
```
min=$(sort -V <<EOF | head -n 1
$CURRENT_VERSION
$VERSION_LIMIT
EOF
)
if [ "$min" -ne "$VERSION_LIMIT" ]; then
  echo "too low"
fi
```
Is there anything cleaner?

Reply:
maybe something like 
```bash
test $(echo "$CURRENT_VERSION\n$VERSION_LIMIT" | sort -V | head -n1) = "$VERSION_LIMIT"
```